contrib/babelfishpg_tsql/sql/sys_procedures.sql

```diff
@@ -202,6 +202,10 @@ CREATE OR REPLACE PROCEDURE sys.sp_droprolemember(IN "@rolename" sys.SYSNAME, IN
 AS 'babelfishpg_tsql', 'sp_droprolemember' LANGUAGE C;
 GRANT EXECUTE on PROCEDURE sys.sp_droprolemember(IN sys.SYSNAME, IN sys.SYSNAME) TO PUBLIC;
 
+CREATE OR REPLACE PROCEDURE sys.sp_execute_postgresql(IN "@extensionStmt" sys.nvarchar)
```

Comment:
Even though we only support extension stmts at the moment, let's try to be more generic to avoid future changes, e.g., `@postgresStmt` instead of `@extensionStmt`.